src/Validation/Validation.php

```diff
@@ -904,11 +904,14 @@ public static function range($check, $lower = null, $upper = null)
      * @param string $check Value to check
      * @param bool $strict Require URL to be prefixed by a valid scheme (one of http(s)/ftp(s)/file/news/gopher)
      * @return bool Success
+     * @link https://tools.ietf.org/html/rfc3986
      */
     public static function url($check, $strict = false)
     {
         static::_populateIp();
-        $alpha = '0-9\p{L}\p{N}';
+
+        $emoji = '\x{1F190}-\x{1F9EF}';
+        $alpha = '0-9(\p{L}\p{N}' . $emoji;
```

Comment:
Is the addition of `(` in the regex intentional? It wasn't there earlier.

Reply:
Nice catch!

Reply:
I can fix that before merging.